app/containers/ViewQuestion/ContentHeader.jsx

```diff
@@ -146,6 +205,69 @@ const ContentHeader = props => {
             <FormattedMessage {...messages.voteToDelete} />
           </Button>
 
+          {questionData.user === userInfo.user && (
+            <DropdownBox>
```

Comment:
Sharing dropdown needs to be a separate component.

Reply:
added separate component